### ISSUE TITLE
Trainset 이후 validation 할 때 진행되지 않음

### ISSUE THREAD

**Issue by JaeungHyun** [openspeech-team/openspeech#159]
## Environment info
- Platform: Linux 5.15.0-25-generic 
- Python version: 3.9
- PyTorch version (GPU?): 1.10
- Using GPU in script?: 3090ti

  
## Information

Model I am using (ListenAttendSpell, Transformer, Conformer ...): LAS

The problem arises when using:
* [ ] my own modified scripts: (give details below)
  | Name      | Type                 | Params
---------------------------------------------------
0 | criterion | CrossEntropyLoss     | 0
1 | encoder   | LSTMEncoder          | 15.0 M
2 | decoder   | LSTMAttentionDecoder | 20.8 M
---------------------------------------------------
35.8 M    Trainable params
0         Non-trainable params
35.8 M    Total params
143.261   Total estimated model params size (MB)
Epoch 0:  98%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████▋    | 25000/25469 [2:09:41<02:25,  3.21it/s, loss=0.0378, v_num=0, train_loss=0.0575, train_wer=0.209, train_cer=0.128]
Validation: 0it [00:00, ?it/s]

학습을 진행할때 위 상태에서 더 이상 학습도 안되고 가만있고

CTC joint las 학습할때도 마찬가지입니다

`def _generate_manifest_files(self, manifest_file_path: str) -> None:
        r"""
        Generate KsponSpeech manifest file.
        Format: AUDIO_PATH [TAB] TEXT_TRANSCRIPTS [TAB] NUMBER_TRANSCRIPT
        """
        # train_valid_audio_paths, train_valid_transcripts = preprocess(
        #     self.configs.dataset.dataset_path, self.configs.dataset.preprocess_mode
        # )
        # test_audio_paths, test_transcripts = preprocess_test_data(
        #     self.configs.dataset.test_manifest_dir, self.configs.dataset.preprocess_mode
        # )
        audio_paths, transcripts = preprocess(
            self.configs.dataset.dataset_path, self.configs.dataset.preprocess_mode
        )
        # train_test_split(audio_paths, transcripts)
        # train_valid_audio_paths, test_audio_paths, train_valid_transcripts, test_transcripts = train_test_split(X, y, test_size=1, random_state=42)

        # audio_paths = train_valid_audio_paths + test_audio_paths
        # transcripts = train_valid_transcripts + test_transcripts

        if self.configs.tokenizer.unit == 'kspon_character':
            generate_character_labels(transcripts, self.configs.tokenizer.vocab_path)
            generate_character_script(audio_paths, transcripts, manifest_file_path, self.configs.tokenizer.vocab_path)`

manifest 파일 만들때 보니까 

Trainset Validset 한번에 불러와서 숫자대로 자르시던데 

aihub에서 제공하는 유소아명령어셋이 validation 데이터가 정확하지 않아서 일단 학습용 데이터로만 불러와서 알아서 자르게 했거든요

거기다 validation 데이터가 없으면 바로 넘어가야할건데 넘어가지 않다보니 

다른부분을 제가 확인해봐야하는지 궁금합니다.

**Comment by sooftware** [comment]
음 manifest 파일 그대로 한번 붙여주실래요?  
형식이 다르면 학습이 당연히 안 될 거에요.
커스텀으로 세팅하신 파일 전체 다 보여주세요.

**Comment by JaeungHyun** [comment]
엇.. 데이터로더가 cpu 1코어만 굴리는게 이상해서 재부팅해보았는데 잘 작동하네요
이슈는 닫도록 하겠습니다
